admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -255,7 +255,7 @@ export default class OrderProductRenderer {
   toggleProductAddNewInvoiceInfo() {
     $(OrderViewPageMap.productAddNewInvoiceInfo).toggleClass(
       'd-none',
-      parseInt($(OrderViewPageMap.productAddInvoiceSelect).val(), 10) === 0,
+      $(OrderViewPageMap.productAddInvoiceSelect).length === 0,
     );
```

Comment:
We must check the existence of the invoice selector instead of its value, because the invoice selector doesn't exist when no invoice exists for this order.